src/uu/du/src/du.rs

```diff
@@ -720,6 +743,35 @@ Try '{} --help' for more information.",
     0
 }
 
+struct Threshold(Option<u64>, bool);
```

Comment:
I had to think for a minute about what the `bool` meant. Maybe it makes sense to make this an `enum` like this to provide a bit more context?
```rust
enum Threshold {
    Less(u64),
    Greater(u64),
}
```

Reply:
Good suggestion - I'll give this a shot.